### ISSUE TITLE
Register Forbidden after production build by webpack 4.x.x

### ISSUE THREAD

**Issue by crowmw** [onsip/SIP.js#556]
Hi!
I update my webpack from 3.x.x up to version 4.8.1
All config is the same.

Now i get `SIP/2.0 403 Forbidden` everytime when i want to register client to server.

Server says it's because wrong password. But when i debug code in chrome dev tools the password looks ok.

I created sample project with the same problem:
https://github.com/crowmw/sip-js-test

I think it can be simillar to:
https://github.com/twilio/twilio-video.js/issues/50#issuecomment-279885506

Any help?

**Comment by crowmw** [comment]
`
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | configuration parameters after validation:
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · viaHost: "uok67o1qfp9m.invalid"
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · uri: sip:388141@server-address.com
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · wsServers: [{"ws_uri":"wss://server-address.com:5066","sip_uri":"<sip:server-address.com:5066;transport=ws;lr>","weight":0,"status":0,"scheme":"WSS"}]
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · custom: {}
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · displayName: ""
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · password: NOT SHOWN
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · registerExpires: 600
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · register: false
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · registrarServer: sip:server-address.com
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · wsServerMaxReconnection: 3
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · wsServerReconnectionTimeout: 4
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · connectionRecoveryMinInterval: 2
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · connectionRecoveryMaxInterval: 30
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · keepAliveInterval: 0
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · extraSupported: []
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · usePreloadedRoute: false
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · userAgentString: "webRTC 1.0 systell"
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · noAnswerTimeout: 60000
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · traceSip: true
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · hackViaTcp: false
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · hackIpInContact: false
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · hackWssInTransport: false
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · hackAllowUnregisteredOptionTags: false
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · sessionDescriptionHandlerFactoryOptions: {"constraints":{"audio":{"deviceId":{}},"video":false},"peerConnectionOptions":{"iceCheckingTimeout":550}}
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · contactName: "0b1p0von"
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · contactTransport: "ws"
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · forceRport: false
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · autostart: true
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · autostop: true
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · rel100: "none"
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · dtmfType: "info"
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · replaces: "none"
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · sessionDescriptionHandlerFactory: function(e,t,s){return new i(e,t,s)}
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · authenticationFactory: undefined
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · allowLegacyNotifications: false
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · allowOutOfDialogRefers: false
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · authorizationUser: "388141"
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · instanceId: "d158899c-730f-4dcb-8171-06e73a577d41"
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · sipjsId: "12i9p"
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · hostportParams: "server-address.com"
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | user requested startup...
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | connecting to WebSocket wss://server-address.com:5066
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | unable to send message, WebSocket is not open
t.print @ sip.min.js:38
t.(anonymous function) @ sip.min.js:38
i.(anonymous function) @ sip.min.js:38
send @ sip.min.js:38
s.send @ sip.min.js:38
send @ sip.min.js:38
send @ sip.min.js:38
register @ sip.min.js:38
register @ sip.min.js:38
s @ index.js:37
document.getElementById.addEventListener @ index.js:57
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.transaction.nict | transport error occurred, deleting non-INVITE client transaction z9hG4bK5165596
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | WebSocket wss://server-address.com:5066 connected
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | connection state set to 0
sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | sending WebSocket message:

REGISTER sip:server-address.com SIP/2.0
Via: SIP/2.0/WSS uok67o1qfp9m.invalid;branch=z9hG4bK6867437
Max-Forwards: 70
To: <sip:388141@server-address.com>
From: <sip:388141@server-address.com>;tag=s2a96dq69r
Call-ID: gpfjftt0ijsnpmtjn9to57
CSeq: 8889 REGISTER
Expires: 60
Contact: <sip:0b1p0von@uok67o1qfp9m.invalid;transport=ws>;reg-id=1;+sip.instance="<urn:uuid:d158899c-730f-4dcb-8171-06e73a577d41>";expires=600
Allow: ACK,CANCEL,INVITE,MESSAGE,BYE,OPTIONS,INFO,NOTIFY,REFER
Supported: path, gruu, outbound
User-Agent: webRTC 1.0 systell
Content-Length: 0



sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | received WebSocket text message:

SIP/2.0 401 Unauthorized
Via: SIP/2.0/WSS uok67o1qfp9m.invalid;branch=z9hG4bK6867437;received=10.213.1.54;rport=54077
From: <sip:388141@server-address.com>;tag=s2a96dq69r
To: <sip:388141@server-address.com>;tag=pH792K8mBt3XN
Call-ID: gpfjftt0ijsnpmtjn9to57
CSeq: 8889 REGISTER
User-Agent: FreeSWITCH
Allow: INVITE, ACK, BYE, CANCEL, OPTIONS, MESSAGE, INFO, UPDATE, REGISTER, REFER, NOTIFY, PUBLISH, SUBSCRIBE
Supported: timer, path, replaces
WWW-Authenticate: Digest realm="server-address.com", nonce="37075ec3-a118-4e53-a7b1-c4f77408381f", algorithm=MD5, qop="auth"
Content-Length: 0



sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | sending WebSocket message:

REGISTER sip:server-address.com SIP/2.0
Via: SIP/2.0/WSS uok67o1qfp9m.invalid;branch=z9hG4bK7941278
Max-Forwards: 70
To: <sip:388141@server-address.com>
From: <sip:388141@server-address.com>;tag=s2a96dq69r
Call-ID: gpfjftt0ijsnpmtjn9to57
CSeq: 8890 REGISTER
Authorization: Digest algorithm=MD5, username="388141", realm="server-address.com", nonce="37075ec3-a118-4e53-a7b1-c4f77408381f", uri="sip:server-address.com", response="bdc1c01ed62996c119885109a20708dd", qop=auth, cnonce="iskkflofk36b", nc=00000001
Expires: 60
Contact: <sip:0b1p0von@uok67o1qfp9m.invalid;transport=ws>;reg-id=1;+sip.instance="<urn:uuid:d158899c-730f-4dcb-8171-06e73a577d41>";expires=600
Allow: ACK,CANCEL,INVITE,MESSAGE,BYE,OPTIONS,INFO,NOTIFY,REFER
Supported: path, gruu, outbound
User-Agent: webRTC 1.0 systell
Content-Length: 0



sip.min.js:38 Mon May 14 2018 15:49:07 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | received WebSocket text message:

SIP/2.0 403 Forbidden
Via: SIP/2.0/WSS uok67o1qfp9m.invalid;branch=z9hG4bK7941278;received=10.213.1.54;rport=54077
From: <sip:388141@server-address.com>;tag=s2a96dq69r
To: <sip:388141@server-address.com>;tag=Qt024eSr82SgH
Call-ID: gpfjftt0ijsnpmtjn9to57
CSeq: 8890 REGISTER
User-Agent: FreeSWITCH
Allow: INVITE, ACK, BYE, CANCEL, OPTIONS, MESSAGE, INFO, UPDATE, REGISTER, REFER, NOTIFY, PUBLISH, SUBSCRIBE
Supported: timer, path, replaces
Content-Length: 0
`

**Comment by egreenmachine** [comment]
I don't see anything explicitly wrong. Can you provide logs from the working version of webpack?

**Comment by seanbright** [comment]
Running into this as well with Angular CLI 6.0.1. Guessing that it has something to do with UglifyJS optimizing something away during bundling.

**Comment by crowmw** [comment]
```
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | configuration parameters after validation:
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · viaHost: "8eahq78t8eh2.invalid"
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · uri: sip:388141@server-address.com.pl
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · wsServers: [{"ws_uri":"wss://server-address.com.pl:5066","sip_uri":"<sip:server-address.com.pl:5066;transport=ws;lr>","weight":0,"status":0,"scheme":"WSS"}]
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · custom: {}
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · displayName: ""
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · password: NOT SHOWN
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · registerExpires: 600
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · register: false
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · registrarServer: sip:server-address.com.pl
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · wsServerMaxReconnection: 3
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · wsServerReconnectionTimeout: 4
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · connectionRecoveryMinInterval: 2
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · connectionRecoveryMaxInterval: 30
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · keepAliveInterval: 0
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · extraSupported: []
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · usePreloadedRoute: false
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · userAgentString: "webRTC 1.0 systell"
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · noAnswerTimeout: 60000
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · traceSip: true
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · hackViaTcp: false
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · hackIpInContact: false
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · hackWssInTransport: false
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · hackAllowUnregisteredOptionTags: false
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · sessionDescriptionHandlerFactoryOptions: {"constraints":{"audio":{"deviceId":{}},"video":false},"peerConnectionOptions":{"iceCheckingTimeout":550}}
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · contactName: "s7mrp9bp"
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · contactTransport: "ws"
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · forceRport: false
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · autostart: true
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · autostop: true
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · rel100: "none"
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · dtmfType: "info"
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · replaces: "none"
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · sessionDescriptionHandlerFactory: function(e,t,s){return new i(e,t,s)}
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · authenticationFactory: undefined
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · allowLegacyNotifications: false
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · allowOutOfDialogRefers: false
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · authorizationUser: "388141"
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · instanceId: "7fd850ec-08fc-47a7-8118-7e6482e29a09"
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · sipjsId: "b7a0h"
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | · hostportParams: "server-address.com.pl"
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | user requested startup...
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | connecting to WebSocket wss://server-address.com.pl:5066
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | unable to send message, WebSocket is not open
i.print @ sip.min.js:38
i.(anonymous function) @ sip.min.js:38
t.(anonymous function) @ sip.min.js:38
send @ sip.min.js:38
s.send @ sip.min.js:38
send @ sip.min.js:38
send @ sip.min.js:38
register @ sip.min.js:38
register @ sip.min.js:38
call @ index.js:37
document.getElementById.addEventListener @ index.js:57
sip.min.js:38 Tue May 15 2018 08:44:28 GMT+0200 (Środkowoeuropejski czas letni) | sip.transaction.nict | transport error occurred, deleting non-INVITE client transaction z9hG4bK3176845
sip.min.js:38 Tue May 15 2018 08:44:29 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | WebSocket wss://server-address.com.pl:5066 connected
sip.min.js:38 Tue May 15 2018 08:44:29 GMT+0200 (Środkowoeuropejski czas letni) | sip.ua | connection state set to 0
sip.min.js:38 Tue May 15 2018 08:44:29 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | sending WebSocket message:

REGISTER sip:server-address.com.pl SIP/2.0
Via: SIP/2.0/WSS 8eahq78t8eh2.invalid;branch=z9hG4bK358554
Max-Forwards: 70
To: <sip:388141@server-address.com.pl>
From: <sip:388141@server-address.com.pl>;tag=n17ggvtp11
Call-ID: cnhcnscd4n1952ld9kta6i
CSeq: 244 REGISTER
Expires: 60
Contact: <sip:s7mrp9bp@8eahq78t8eh2.invalid;transport=ws>;reg-id=1;+sip.instance="<urn:uuid:7fd850ec-08fc-47a7-8118-7e6482e29a09>";expires=600
Allow: ACK,CANCEL,INVITE,MESSAGE,BYE,OPTIONS,INFO,NOTIFY,REFER
Supported: path, gruu, outbound
User-Agent: webRTC 1.0 systell
Content-Length: 0



sip.min.js:38 Tue May 15 2018 08:44:29 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | received WebSocket text message:

SIP/2.0 401 Unauthorized
Via: SIP/2.0/WSS 8eahq78t8eh2.invalid;branch=z9hG4bK358554;received=10.213.1.54;rport=51055
From: <sip:388141@server-address.com.pl>;tag=n17ggvtp11
To: <sip:388141@server-address.com.pl>;tag=6DtH1USeaXytg
Call-ID: cnhcnscd4n1952ld9kta6i
CSeq: 244 REGISTER
User-Agent: FreeSWITCH
Allow: INVITE, ACK, BYE, CANCEL, OPTIONS, MESSAGE, INFO, UPDATE, REGISTER, REFER, NOTIFY, PUBLISH, SUBSCRIBE
Supported: timer, path, replaces
WWW-Authenticate: Digest realm="server-address.com.pl", nonce="54860d69-db47-48a1-bfc2-bd022c8c8c34", algorithm=MD5, qop="auth"
Content-Length: 0



sip.min.js:38 Tue May 15 2018 08:44:29 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | sending WebSocket message:

REGISTER sip:server-address.com.pl SIP/2.0
Via: SIP/2.0/WSS 8eahq78t8eh2.invalid;branch=z9hG4bK9133145
Max-Forwards: 70
To: <sip:388141@server-address.com.pl>
From: <sip:388141@server-address.com.pl>;tag=n17ggvtp11
Call-ID: cnhcnscd4n1952ld9kta6i
CSeq: 245 REGISTER
Authorization: Digest algorithm=MD5, username="388141", realm="server-address.com.pl", nonce="54860d69-db47-48a1-bfc2-bd022c8c8c34", uri="sip:server-address.com.pl", response="b85ee0b35fd03547ca3dcaa3cd4c1133", qop=auth, cnonce="6ugqrh5buv7t", nc=00000001
Expires: 60
Contact: <sip:s7mrp9bp@8eahq78t8eh2.invalid;transport=ws>;reg-id=1;+sip.instance="<urn:uuid:7fd850ec-08fc-47a7-8118-7e6482e29a09>";expires=600
Allow: ACK,CANCEL,INVITE,MESSAGE,BYE,OPTIONS,INFO,NOTIFY,REFER
Supported: path, gruu, outbound
User-Agent: webRTC 1.0 systell
Content-Length: 0



sip.min.js:38 Tue May 15 2018 08:44:29 GMT+0200 (Środkowoeuropejski czas letni) | sip.transport | received WebSocket text message:

SIP/2.0 200 OK
Via: SIP/2.0/WSS 8eahq78t8eh2.invalid;branch=z9hG4bK9133145;received=10.213.1.54;rport=51055
From: <sip:388141@server-address.com.pl>;tag=n17ggvtp11
To: <sip:388141@server-address.com.pl>;tag=7pKa3paj75mDc
Call-ID: cnhcnscd4n1952ld9kta6i
CSeq: 245 REGISTER
Contact: <sip:s7mrp9bp@8eahq78t8eh2.invalid;transport=ws>;expires=60
Date: Tue, 15 May 2018 06:44:22 GMT
User-Agent: FreeSWITCH
Allow: INVITE, ACK, BYE, CANCEL, OPTIONS, MESSAGE, INFO, UPDATE, REGISTER, REFER, NOTIFY, PUBLISH, SUBSCRIBE
Supported: timer, path, replaces
Content-Length: 0
```

**Comment by seanbright** [comment]
@egreenmachine would you be willing to accept a PR which removes the self-rolled `calculateMD5()` in favor of `crypto-js/md5` here?

**Comment by james-criscuolo** [comment]
Hi Sean,
I was talking to @egreenmachine about this offline, I was going to go add a test and ideally that test would break when I updated our library to webpack 4. On the other hand, I don't think this md5 function is necessarily something we want to support going forward, so yes, we will accept a PR replacing it, and crypto-js/md5 looks great.

Thanks,
James

**Comment by seanbright** [comment]
@james-criscuolo how do I lobby for a 0.10.1 (or 0.11.0)? 😀

**Comment by james-criscuolo** [comment]
I'm looking at upgrading our build process to use webpack 4 today, then I will see if @egreenmachine has anything before we update. We may be waiting on another PR.
